#!/usr/bin/env python
PACKAGE = "ohmni_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("trigger_all_waypoints",   bool_t,   0, "A Boolean parameter",  False)
gen.add("pause_at_next_goal",   bool_t,   0, "A Boolean parameter",  False)
gen.add("cancel_all_goal",   bool_t,   0, "A Boolean parameter",  False)

exit(gen.generate(PACKAGE, "ohmni_nav", "WaypointNavigation"))